hermes-common/src/main/java/pl/allegro/tech/hermes/common/metric/counter/zookeeper/CounterMatcher.java

```diff
@@ -33,6 +39,14 @@ public boolean isTopicPublished() {
         return counterName.startsWith("published.");
     }
 
+    public boolean isTopicThroughput() {
+        return counterName.startsWith("throughput.") && metricParts == TOPIC_METRICS_PARTS;
```

Comment:
WDYT about changing order of operations? (same a func code below)
```suggestion
        return metricParts == TOPIC_METRICS_PARTS && counterName.startsWith("throughput.");
```

Reply:
sure